### DISCUSSION TITLE
Are there permanent links or URLs I could use?

### DISCUSSION THREAD

**Discussion by 0berlehrer** [datenstrom/community#766] [Ask a question] [answered]
When I change the title of a page or in a story, parts of the links to this page change too.
This is a problem for links inside the website and from outside...   404 ;-) 

So 'never change a title' or are there permanent links I could use?

**Comment by schulle4u** [comment, marked as answer]
As a permalink solution you can use the `TitleSlug` option in your [page settings](https://datenstrom.se/yellow/help/how-to-change-the-system#page-settings) to control the page URL. Is this what you're looking for?

**Reply by schulle4u** [reply]
Here is an example page with custom URL slug: 

```
---
Title: About me and my projects
TitleSlug: about
---
Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut 
labore et dolore magna pizza. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris 
nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit 
esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt 
in culpa qui officia deserunt mollit anim id est laborum.

[Made with Datenstrom Yellow](https://datenstrom.se/yellow/).
```

The page is available e. g. as `https://website.com/about/`. Even if you change the `Title` setting using your browser, the URL and file name should still stay the same thanks to the `TitleSlug` setting.

**Reply by 0berlehrer** [reply]
Thanx :-) Yes, the title does not change... but if you change the date (blog entry) then the combined file names changes. And the date is important for sorting the blog entries.

Because in a website/blog internal and external linking is important, in my opinion this is a kind of "bug" (or i have a complete misunderstanding).

Over there, i like the simple concept of yellow!

**Reply by schulle4u** [reply]
Changing date settings shouldn't break your links. It is possible to [use different placeholders](https://github.com/annaesvensson/yellow-blog/blob/main/README.md#settings-placeholders) in the location for new blog entries, but by default the date setting is not part of the location and only used to sort blog pages in the blog-start layout.

**Reply by BogusCurry** [reply]
Hi there ;D
The Link to the [use different placeholders](https://github.com/datenstrom/yellow-extensions/tree/master/source/blog#settings-placeholders) ist dead ;D

Can you fix this ?

Greetings

**Reply by schulle4u** [reply]
Thanks, I've updated the link to the new repository.

**Comment by annaesvensson** [comment]
As an alternative you can use `Redirect` in your page settings. This is probably not what you were looking for, but, it could be quite useful when you found broken links on your website and want to [redirect a page to another page](https://github.com/annaesvensson/yellow-core#how-to-redirect-a-page). Once you have found broken links on your website, add a new page at that exact URL and then redirect it to a suitable replacement. Here's an example:

```
---
Title: Redirect page
Redirect: https://datenstrom.se/yellow/
---
This page is redirected to another page.
```
